src/sqlfluff/core/default_config.cfg

```diff
@@ -145,6 +145,9 @@ wildcard_policy = single
 # Trailing commas
 select_clause_trailing_comma = forbid
 
+[sqlfluff:rules:L039]
+align_alias = False
```

Comment:
For testing purposes, I suggest you _temporarily_ change this to `True`, then commit and push the change. The "Critical Rules Error" test will run `sqlfluff fix` on all of the SQL test files in SQLFluff. This check tends to discover a lot of issues similar to the one I discovered by manual testing.